MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2548,7 +2548,10 @@ public string GetHardwareAcceleratedVideoDecoder(VideoType videoType, MediaStrea
                 {
                     if (Environment.OSVersion.Platform == PlatformID.Win32NT)
                     {
-                        return "-hwaccel dxva2";
+                        if( Environment.OSVersion.Version.Major > 6 || ( Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor > 1 ) )
+                            return "-hwaccel d3d11va";
+                        else
+                            return "-hwaccel dxva2";```
```

Comment:
Suggest changes somehow left my markdown ending charcter in there. That need to go but then approved
```suggestion
                            return "-hwaccel dxva2";
```